dashboard/test/ui/runner.rb

```diff
@@ -25,6 +26,35 @@
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+S3_LOGS_BUCKET = 'cucumber-logs'
+S3_LOGS_PREFIX = rack_env?(:test) ? 'test' : 'development'
```

Comment:
I'm open to suggestions on a better way to split this out; maybe I should just use hostname?  I really want everything that runs on the test machine (re-runs, etc) to get bucketed together under test/ but for people doing UI tests locally to get their own bucket.

Oh!  Or branch name.  That wouldn't be too bad.